endpointmanager/pkg/endpointmanager/fhirendpointinfo.go

```diff
@@ -80,5 +81,35 @@ func (e *FHIREndpointInfo) Equal(e2 *FHIREndpointInfo) bool {
 		return false
 	}
 
+	// validationTest := cmp.Equal(e.Validation, e2.Validation)
```

Comment:
Not sure if this line was meant to be left as a comment, but it looks like it might be something you might have wanted to remove

Reply:
nope you're right it was old code, good catch! it's been updated